tools/ci_build/github/windows/bundle_dlls_gpu_nuget.bat

```diff
@@ -7,21 +7,8 @@ SETLOCAL EnableDelayedExpansion
 FOR /R %%i IN (*.nupkg) do (
    set filename=%%~ni
    IF NOT "!filename:~25,7!"=="Managed" (
-       mkdir runtimes\linux-x64\native
```

Comment:
is this removed section somewhere else now?
makes the name of the batch file kind of odd since there is no longer any processing of dll's by this batch file.